src/Components/RelatedItems/RelatedItems.jsx

```diff
@@ -147,7 +39,6 @@ var RelatedItems = ({current, setCurrentById, getProducts}) => {
         comparison.push([currentFeatures[combinedKeys[i]], combinedKeys[i], comparatorFeatures[combinedKeys[i]]])
       // }
```

Comment:
Two comparison variables in the same document. I though t you were assigning values directly to the STATE comparison which is something to not do and always use the "setting" variable. Further down I see you declared another comparison variable using var. 
Just something to maybe look out for. line 15 const comparison. and line 26 var comparison

Reply:
Good catch- the scoping works out okay in this case, but it does not read well at all. I'll make a note to modify this in the next update. Thanks!